pkg/controllers/vscr/podpolling_reconciler.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright [2021-2023] Open Text.
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This copyright notice looks different then other files. Its missing some blank lines?

Reply:
I see this in other files too (podpolling_reconciler_test.go)

Reply:
Hum I will take a closer look. Mind giving an example of a file that has the "correct" copyright statement?

Reply:
pkg/controllers/vdb/actor.go